resources/views/admin/layouts/header_post.blade.php

```diff
@@ -1,7 +1,7 @@
 <meta charset="utf-8" />
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 <title>
-    {{ $title ?? 'Title' }}
+    {{ isset($title) ? $title : 'Title' }}
```

Comment:
thay đổi này tương tự nhau, em dùng ?? cho dẹp hơn cũng đc 
em check lại các syntax của toán tử để dùng cho pro nhen: https://github.com/anhlp1212/blog_laravel/pull/6/commits/830ffc1aff179e2ba22b4c0ac565bf674cada2ca